modules/layers/package.json

```diff
@@ -40,7 +40,7 @@
   },
   "peerDependencies": {
     "@deck.gl/core": "^8.0.0",
-    "@loaders.gl/core": "^3.0.0",
```

Comment:
@Pessimistress A few peerDependencies are kept at 3.0.0 is that intentional?

Reply:
Yes, our `peerDependencies` are only loosely constrained, usually to the major release. This particular entry is there to satisfy the import lint check. The actual installed version is determined by that in the `depependencies` field.